HW1/P3/P3.py

```diff
@@ -0,0 +1,18 @@
+import findspark
+findspark.init()
+import pyspark
+sc = pyspark.SparkContext(appName='anagrams')
+
+with open('EOWL_words.txt', 'r') as rfile:
+	words = rfile.read().split('\n')
+
+lex_order = lambda s: ''.join(sorted(list(s)))
+
+words_rdd = sc.parallelize(words, 10)
+lex_to_word = words_rdd.map(lambda w: (lex_order(w), w))
```

Comment:
I appreciate the discussion, I was aiming for the function's output to be evident for the name so the reader wouldn't have to parse ''.join(sorted(list(s))) when reading the second line or do any "digging" unless they were interested in implementation details.